examples/specs_integration/src/your_state.rs

```diff
@@ -0,0 +1,44 @@
+use specs::prelude::*;
+use sync::{register_gd_components, Pos};
+
+pub struct YourState {
+    /// This is a useful global container where you can put your junk is later going to be protected
+    /// By a mutex
+    world: World, // Add your stuff here if you like...
+}
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+pub struct ExampleComponent {
+    pub child_id: u32,
+}
+
+impl Component for ExampleComponent {
+    type Storage = VecStorage<Self>;
+}
+
+impl YourState {
+    pub fn new() -> Self {
+        // For more on how to set up a Specs world, be sure to see the specs documentation on
+        //  docs.rs
+        let mut world = World::new();
+
+        // In my project (The Recall Singularity - https://twitter.com/RecallSingular1),
+        //   I register some components in a different crate which provides the game logic.
+        //   Then I add Godot tracking components to the world later using some hooks. So remember
+        //   this stuff is dynamic (yay!)

```

Comment:
This information doesn't seem directly relevant to how the example works. Maybe users should refer to the specs docs for how to use it?

Also, please note that once you contribute this example to this repo, it will be maintained by the community, and may receive contributions from others. The moment this example is merged, it will no longer have a clear "author" figure, but will only be under the collective name of "The godot-rust developers". Thus, any information regarding personal projects should be removed from the example if possible. Thanks for your understanding.